ghost/post-revisions/package.json

```diff
@@ -4,23 +4,28 @@
   "repository": "https://github.com/TryGhost/Ghost/tree/main/packages/post-revisions",
   "author": "Ghost Foundation",
   "private": true,
-  "main": "index.js",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
   "scripts": {
-    "dev": "echo \"Implement me!\"",
-    "test:unit": "NODE_ENV=testing c8 --all --check-coverage --100  --reporter text --reporter cobertura mocha './test/**/*.test.js'",
+    "dev": "tsc --build --watch --preserveWatchOutput",
+    "build": "tsc --build",
+    "prepare": "yarn build",
+    "setup": "yarn build",
+    "test:unit": "NODE_ENV=testing c8 --src src --all --check-coverage --100  --reporter text --reporter cobertura mocha -r ts-node/register './test/**/*.test.ts'",
```

Comment:
`ts-node` means we don't need to add a compile step for tests - I _think_ this is preferable for now